02-microarray/pathway-analysis_microarray_00_intro.Rmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "Pathway Analysis Introduction"
```

Comment:
More general comment - do we need to add this in this pull request? `02-microarray/pathway-analysis_microarray_02_ora.Rmd` is over 400 lines by itself!

Reply:
I see now that this is from a file that was being deleted and that wasn't using "up to date" style guidelines.  In this case, it feels like Josh's point in https://github.com/AlexsLemonade/refinebio-examples/issues/295#issuecomment-708584964 about marking something as a draft would come in handy. This is the first thing I see when I go to review.

Reply:
I'll just remove it entirely from this PR.